plugins/scaffolder-backend/src/lib/templating/SecureTemplater.ts

```diff
@@ -17,8 +17,10 @@
 import { VM } from 'vm2';
 import { resolvePackagePath } from '@backstage/backend-common';
 import fs from 'fs-extra';
+import { JsonValue } from '@backstage/types';
 import { RepoSpec } from '../../scaffolder/actions/builtin/publish/util';
 
+// language=JavaScript
```

Comment:
This is a hint to JetBrains IDEs that the following string should be formatted and highlighted as JavaScript. I think it's easy enough to ignore for non-JetBrains folks, but I'm fine removing it if you guys would prefer.